src/java.base/share/classes/java/security/Security.java

```diff
@@ -702,14 +702,15 @@ static Object[] getImpl(String algorithm, String type, Provider provider,
      *
      * @param key the key of the property being retrieved.
      *
-     * @return the value of the security property corresponding to key.
+     * @return the string value of the security property, or null if there
```

Comment:
I don't think saying "string" is necessary - since this method can only return a String.
Put "null" in code syntax: {@code null}.